addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1531,7 +1530,7 @@ var SnippetEditor = Widget.extend({
      * @param {OdooEvent} ev
      */
     _onSnippetOptionVisibilityUpdate: function (ev) {
-        if (this.willDestroyEditors) {
+        if (this.options.wysiwyg.snippetsMenu && this.options.wysiwyg.snippetsMenu.savingContent) {
```

Comment:
- You are in a snippet editor instance here, there is necessarily a snippetsMenu (one of the widget ancestor).
- I would really prefer you don't check a variable so deep in the widgets like that, I am actually not a fan of `this.options.wysiwyg` in the first place: I would like it but only when we get rid of the trigger_up system as mixing both is just a nightmare :grimacing: But it's ok. Here, instead of `this.options.wysiwyg.snippetsMenu.savingContent`, can we have something `this.options.wysiwyg.isSaving()` -> a method that would check your `savingContent` flag ?